src/app/Timer.ts

```diff
@@ -0,0 +1,30 @@
+export class Timer {
+  private maxTurnTime: number;
+  private turnTimeLeft: number;
+
+  constructor(turnTime = 15) {
+    this.maxTurnTime = turnTime;
+    this.turnTimeLeft = this.maxTurnTime;
+  }
```

Comment:
Consider those changes - it'll save two lines and one variable still doing the same work.
```.ts
// private maxTurnTime: number;
 private turnTimeLeft: number;

 constructor(private maxTurnTime = 15) {
 this.turnTimeLeft = this.maxTurnTime;
 }

Reply:
good point, changed it already :)